packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -72,9 +72,6 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
 
   if (configuration.webGestureHandling != null) {
     options.gestureHandling = configuration.webGestureHandling!.name;
-  } else if (configuration.scrollGesturesEnabled == false ||
```

Comment:
@ditman Is this a breaking change? It doesn't break existing code, but it does change existing behavior.